### PR TITLE
Allow config of ca_file_path

### PR DESCRIPTION
**Pull request by tombruijn**
Needed to be configured in the agent as well so add a config option for
it to be set to the environment.

It only sets non-default values to the environment so that the default
behavior we have now is not changed.

Agent PRs:
https://github.com/appsignal/appsignal-agent/pull/183
https://github.com/appsignal/appsignal-agent/pull/184

Part of https://github.com/appsignal/appsignal-agent/issues/181

### CODE REVIEW

**Comment by thijsc** [comment]
👍

File: lib/appsignal/config.rb [hunk 1]
```diff
@@ -127,6 +129,10 @@ def write_to_environment
       ENV['APPSIGNAL_ENABLE_MINUTELY_PROBES']       = config_hash[:enable_minutely_probes].to_s
       ENV['APPSIGNAL_HOSTNAME']                     = config_hash[:hostname].to_s
       ENV['APPSIGNAL_PROCESS_NAME']                 = $0
+
+      unless config_hash[:ca_file_path] == DEFAULT_CONFIG[:ca_file_path]
```
Comment (matsimitsu) [inline lib/appsignal/config.rb]: Should we maybe check if the file exists?

Comment (tombruijn) [inline lib/appsignal/config.rb]: Probably, but it will probably also have to be added to the agent. It looks like the agent's new http library doesn't output an error if there's a problem with the file.

Comment (tombruijn) [inline lib/appsignal/config.rb]: When a file does not exist or is not readable the Ruby process would crash. Now I don't see us using the Transmitter a lot (the only place this is used in Ruby), but i added a check for existence and readability, and if this not correct it ignores the setting and logs a warning.

would that work?
I didn't want to do this check when the config is loaded, because we do want to know what setting a user is using.

How we catch this on the agent side is another thing, depends on the http library implementation

File: lib/appsignal/transmitter.rb [hunk 2]
```diff
@@ -42,7 +41,7 @@ def uri
     end
 
     def transmit(payload)
-      Appsignal.logger.debug "Transmitting payload to #{uri}"
+      config.logger.debug "Transmitting payload to #{uri}"
```
Comment (thijsc) [inline lib/appsignal/transmitter.rb]: Why this change?
